test/unit/css.js

```diff
@@ -1738,9 +1738,13 @@ QUnit.testUnlessIE( "css(--customProperty)", function( assert ) {
 		"    .test__customProperties {\n" +
 		"        --prop1:val1;\n" +
 		"        --prop2: val2;\n" +
-		"        --prop3:val3 ;\n" +
-		"        --prop4:\"val4\";\n" +
-		"        --prop5:'val5';\n" +
+		"        --prop3:   val3;\n" +
+		"        --prop4:val4 ;\n" +
+		"        --prop5:val5   ;\n" +
+		"        --prop6: val6 ;\n" +
+		"        --prop7:   val7   ;\n" +
+		"        --prop8:\"val8\";\n" +
+		"        --prop9:'val9';\n" +
```

Comment:
Should there be test cases with [CSS whitespace](https://www.w3.org/TR/css-syntax-3/#whitespace) other than U+0020 SPACE?

Reply:
Good point. That also makes me think, we should perhaps replace CSS whitespace instead of calling `.trim()`, shouldn't we?

Reply:
Yes, definitely.

Reply:
I'd be good to have a test case with some Unicode whitespace that is not CSS whitespace to confirm the new planned behavior.